### ISSUE TITLE
BetterBookmarks doesn't open the cached bookmarks on open

### ISSUE THREAD

**Issue by dusk125** [dusk125/sublime-betterbookmarks#3]

**Comment by Kristinita** [comment]
### 1. Summary

My bookmarks not saved for me, if I close file with bookmarks.

### 2. Expected behavior

Saving bookmarks after file closing.

### 3. Actual behavior

I can not see bookmarks after file re-opening.

### 4. Settings

My `User\BetterBookmarks.sublime-settings` file

```json
{
    "cleanup_empty_cache_on_close": false,
    "layer_icons":
    {
        "bookmarks":
        {
            "icon": "bookmark",
            "scope": "string"
        },
        "functions":
        {
            "icon": "Packages/Better Bookmarks/icons/function.png",
            "scope": "comment"
        },
        "todos":
        {
            "icon": "Packages/Better Bookmarks/icons/todo.png",
            "scope": "comment"
        }
    }
}
```

### 4. Steps to reproduce

The problem is reproduced for me in a version of Sublime Text without plugins and user settings.

I install BetterBookmarks → I run command `better_bookmarks_mark_line` → I close file → I re-open file → I can not see my bookmark.

### 5. Environment

**Operating system and version:**
Windows 32-bit 10.0.14393
**Sublime Text:**
Build 3126


Thanks.

**Comment by dusk125** [comment]
This problem is fixed by version 1.1. Also thanks for the awesome bug report!
